giotto/images/preprocessing.py

```diff
@@ -0,0 +1,113 @@
+"""Image preprocessing."""
+# License: Apache 2.0
+
+import numbers
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+from joblib import Parallel, delayed, effective_n_jobs
+from sklearn.utils import gen_even_slices
+from sklearn.utils.validation import check_is_fitted, check_array
+from ..utils.validation import validate_params
+
+
+class Binarizer(BaseEstimator, TransformerMixin):
+    """Transformer returning a collection of binary images from an input
+    collection of 2D or 3D boolean images.
+
+    Parameters
+    ----------
+    threshold : float, default: 0.5
+        Percentage of the maximum pixel value of the collection of image from
```

Comment:
I'm not sure I understand what is meant by "the collection of image" here: are you saying that given a collection of images, define a threshold relative to the pixel(s) that have the maximum value in the collection?